<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWindowReactorCatalog</class>
 <widget class="QDialog" name="SearchWindowReactorCatalog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search Phrase:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SWReactorCatalog_SP_lineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Search By:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="SWReactorCatalog_SB_comboBox">
       <item>
        <property name="text">
         <string>ID</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Name</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JobID</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="SWReactorCatalog_tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="new_pushButton">
       <property name="text">
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="edit_pushButton">
       <property name="text">
        <string>Edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_pushButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>SWReactorCatalog_tableView</zorder>
  <zorder></zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
